lib/checks/visibility/hidden-content.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "hidden-content",
+  "evaluate": "hidden-content.js",
+  "metadata": {
+    "impact": "minor",
+    "messages": {
+      "pass": "Page does not have any suspicious hidden content.",
+      "fail": "Manually inspect hidden content on this page.",
+      "undefined": "One or more elements on this page are hidden from assistive technology. You should verify these elements manually."
```

Comment:
We should add a little more detail so people know what they're looking for to resolve it. Perhaps something like this? 

> One or more elements on this page are hidden from assistive technology. You should verify it was intentional for these elements with manual debugging.